native/src/components/Pagination.tsx

```diff
@@ -16,12 +16,12 @@ const DotsContainer = styled.View`
 `
 
 const Dot = styled(Pressable)<{ isActive: boolean }>`
-  width: 10px;
-  height: 10px;
+  width: 12px;
+  height: 12px;
   border-radius: 5px;
```

Comment:
:wrench: Please adjust the radius as well, otherwise the dots are not circles as shown in the design
```suggestion
  border-radius: 6px;
```